tests/modules/nf-core/jasminesv/nextflow.config

```diff
@@ -0,0 +1,13 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: "test_jasminesv_iris:JASMINESV" {
```

Comment:
Would it make sense to also test for additional iris args?

Reply:
Yeah that would be a good idea! Thanks for the suggestion, I'll look into it!